### ISSUE TITLE
Is this library active? it not compatible with last versions of moq... 

### ISSUE THREAD

**Issue by arielorvits** [structuremap/StructureMap.AutoMocking#5]
"Unable to find a version of 'Moq' that is compatible with 'structuremap.automocking.moq 4.0.0.315 constraint: Moq (= 4.0.10827)'."

**Comment by jeremydmiller** [comment]
I should have deleted this repo. All the code for this is in the main StructureMap repo. I've been wanting to discontinue the separate packages forever. All the auto-mocking stuff is in the main lib except for the mocking library specific adapter. Find the MoqAutoMocker class in the main StructureMap repo and copy/paste that code into yours and you should be good to go.

**Comment by arielorvits** [comment]
Thanks!
I didn't found this on the main structureMap, and also needed to copy 3 classes (to compile the code):
MoqFactory
MoqServiceLocator
MoqAutoMocker
(from StructureMap.AutoMocking repository).

and now it is working.

why wont this code will be released officially as part of updated structuremap package?

**Comment by jeremydmiller** [comment]
Because I no longer believe that it's useful to support this library. Everything takes up time. Keeping up with Moq updates, pushing nugets, answering questions. This is just a place where I think I can jettison some OSS workload.
 

    On Tuesday, March 14, 2017 3:48 AM, arielorvits <notifications@github.com> wrote:
 

 Thanks!
I didn't found this on the main structureMap, and also needed to copy 3 classes (to compile the code):
MoqFactory
MoqServiceLocator
MoqAutoMocker
(from StructureMap.AutoMocking repository).and now it is working.why wont this code will be released officially as part of updated structuremap package—

**Comment by arielorvits** [comment]
It's your decision, but I just started using this and I think it is great. and very very useful! saving my time! thanks again 👍

**Comment by digitaldias** [comment]
I can't live without this one! structuremap.AutoMocking.Moq is my life and blood!

**Comment by arielorvits** [comment]
@digitaldias  I now switched to [autoFixture](https://github.com/AutoFixture/AutoFixture) which do the same and maintained as nuget package

**Comment by jeremydmiller** [comment]
@digitaldias You can still happily use the auto mocking if you'll just copy/paste the Moq adapter bit of it into your own code. The core auto-mocking support is in StructureMap itself. I use it a little bit w/ NSubstitute in newer projects.

@arielorvits You know that the original AutoFixture author just walked away from the project, right?

**Comment by arielorvits** [comment]
@jeremydmiller yes, but still it has big community, and it is more comfortable to use nuget package than copy paste code in my opinion. especially on upgrading other packages.

**Comment by jeremydmiller** [comment]
Maybe, but you're setting yourself up for diamond dependency hell. Mostly though, I just don't think that what SM's automocking or AutoFixture does is all that valuable these days.

**Comment by arielorvits** [comment]
using nunit, moq. when I want separate SetUps for groups of test, so creating some testFixtures for single class (with 10 dependencies for example).

if I need explicitly register each dependency in all testFixtures it'll spent my time. using SM's automocking or AutoFixture I setup just 1-2 dependencies in each testFixture.

**Comment by Arithmomaniac** [comment]
See https://github.com/ChuckBryan/StructureMap.AutoMocking.Moq
